### DISCUSSION TITLE
Group节点不能双击编辑文本

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1202] [Q&A] [answered]
我想在定义的Group节点中还是能够双击进入编辑text，但是发现双击无法实现，但是右键菜单的编辑文本又可以进行编辑，我通过监听鼠标的双击事件，去调用lf的editText方法，还是无效，请问这种应该怎么实现呀？！！

**Comment by paiap** [comment]
还有一个问题，不知道是不是bug，就是当text为空的时候，有时候双击编辑文本时，文本的位置固定在了画布的最左上方，有值的时候是正常的。

**Comment by towersxu** [comment, marked as answer]
Group节点默认关闭了双击文案编辑，你可以自定义Group节点再覆盖回来。图标group也是一个节点，所以你可以参考自定义节点修改。

**Reply by paiap** [reply]
好的，我调用了editText，它可以了！
